pyproject.toml

```diff
@@ -41,6 +41,7 @@ classifiers = [
 
 dependencies = [
   "bluesky",  # Bluesky framework
+  "bluesky-kafka",  # Bluesky-kafka integration
```

Comment:
msgpack is probably brought in from bluesky-kafka but do we need to explicitly add it here?

Reply:
Added `msgpack-numpy` as an explicit dep